liberty-maven-plugin/src/it/kernel-install-feature-test/install-features-override-server-dir-it/src/test/java/net/wasdev/wlp/test/features/it/InstallFeaturesOverrideServerDirTest.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corporation 2022.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package net.wasdev.wlp.test.feature.it;
+
+import org.junit.Test;
+
+public class InstallFeaturesOverrideServerDirTest extends BaseInstallFeature {
+    
+    @Test
+    public void testInstalledFeatures() throws Exception {
+        assertNotInstalled("servlet-4.0");
+        assertInstalled("jsp-2.3");
+        assertInstalled("servlet-3.1");
```

Comment:
why is servlet-3.1 generated? Is it a dependency for jsp-2.3?
Maybe add a comment to further explain this test.

Reply:
The GH action wasn't actually running this test - fixed now. Updated comments as well